Dockerfile

```
@@ -0,0 +1,106 @@
+# git is already present in rust:*-bookworm and is required: the templar-*
+# dependencies below are pulled from github.com/Templar-Protocol/contracts
+# via git Cargo dependencies, so this stage also needs network access.
+#
+# nodejs + npm: templar-gateway-oracle-updates-dispatch (a direct dependency,
+# used here only for its Pyth source - see src/service.rs's
+# `.with_pyth_source`) unconditionally depends on templar-redstone-bridge,
+# whose build.rs unconditionally shells out to `npm install`/`build`/`bundle`
+# to produce a JS bundle embedded in the compiled binary. Same requirement as
+# services/blockchain-gateway/Dockerfile in the backend monorepo, which
+# depends on the same crate.
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.15.1)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)